packages/reporter/src/settings/index.ts

```diff
@@ -0,0 +1,40 @@
+import "dotenv/config"
+
+export const settings = {
+  contractAddresses: {
+    goerli: {
+      AMBReporter: "0xedc0b1d3de4496e0d917af42f29cb71eb2982319",
+      SygmaReporter: "0x2f96d347c932ac73b56e9352ecc0707e25173d88",
+    },
+    mainnet: {},
+    gnosis: {
+      AMBAdapter: "0x01268DB05965CeAc2a89566c42CD550ED7eE5ECD",
+      SygmaAdapter: "0x9AD7a6f4FDA8247cC0bF5932B68c5b619937dB15",
+      TelepathyLightClient: "0x34b5378DE786389a477b40dD710812c250185f83",
+      TelapathyAdapter: "0x2f1E51a2763FB67fe09971Fd8d849716137A3357",
+    },
+  },
+  reporterController: {
```

Comment:
`reporterControllers` not `reporterController` .

Variables naming is important in order to produce an easy to understand code.

Moreover i would use the uppercase letter on each reporter controller.

ex: 
`sygmaReporterController` -> `SygmaReporterController`